regression_report.py

```diff
@@ -131,8 +133,12 @@ def checkBenchmark(args, exe, benchmark):
     median_trend = 0
     if median != 0:
         median_trend = recent_median * 100 / median - 100
-    if median_trend < args.medianTrendThreshold:
-        # regression detected, print warning
+
+    # regression detected, print warning
+    if median_trend < args.medianTrendThreshold and lessIsbBetter is False:
+        print "<%s|%s> median=%s recent_median=%s" % (urlToBenchmark, benchmark, median, recent_median)
+
+    if median_trend > (-1 * args.medianTrendThreshold) and lessIsbBetter:
```

Comment:
These two lines don't seem very readable to me.
How about extracting it into a function e.g. `isThresholdReached` returning a boolean?
That would also eliminate code duplication.

Reply:
👍Good point, extracted it into `isThresholdReached`.